lib/app/features/feed/stories/views/pages/story_record_page.dart

```diff
@@ -35,12 +38,24 @@ class StoryRecordPage extends HookConsumerWidget {
 
     final isCameraReady = cameraState is CameraReady;
 
-    ref.listen<StoryCameraState>(storyCameraControllerProvider, (_, next) {
+    ref.listen<StoryCameraState>(storyCameraControllerProvider, (_, next) async {
       if (next is StoryCameraSaved && context.mounted) {
-        StoryPreviewRoute(
-          path: next.file.path,
-          mimeType: next.file.mimeType,
-        ).push<void>(context);
+        var filePath = await ref.read(assetFilePathProvider(next.file.path).future);
```

Comment:
A duplication from `idle_camera_preview.dart`. It makes sense to move this code into a provider as written above

Reply:
fixed